API/app.test.js

```diff
@@ -107,6 +106,72 @@ describe('GET /api/reviews/:review_id', () => {
           });
       });
 });
-    
 
-      
\ No newline at end of file
+describe('POST /api/reviews/:review_id/comments', () => {
+    it('responds with 201: request body accepts an object with username and body property, returns posted comment', () => {
+        const review_id = 1
+        const newComment = {
+            username: "dav3rid",
+            body: "hahahehe"
+        }
+        return request(app) 
+            .post(`/api/reviews/${review_id}/comments`)
+            .send(newComment)
+            .expect(201)
+            .then(({body}) => {
+                expect(body).toMatchObject({
+                    comment_id: 7,
+                    body: 'hahahehe',
+                    review_id: 1,
+                    author: "dav3rid",
+                    votes: 0, 
+                    created_at: expect.any(String)
+                })
+            })
+    })
+    it("should responsd with 404 and review not found", () => {
+        const review_id = 50;
+        const newComment = {
+            username: "dav3rid",
+            body:'hehehohoegg'
+        }
+        return request(app)
+        .post(`/api/reviews/${review_id}/comments`)
+        .send(newComment)
+        .expect(404)
+        .then(( body) => {
+            expect(body.text).toEqual(`cannot find review_id`);
+            });
+    }) 
+    it("should response with 400 and bad request", () => {
+        const review_id = 1
+        const newComment = {
+            username: "dav3rid",
+            body:'hehehohoegg'
+        }
+        return request(app)
+          .post(`/api/reviews/egg/comments`)
+          .send(newComment)
+          .expect(400)
+          .then(( body) => {
+            expect(body.text).toEqual(`bad request`);
```

Comment:
need another 400 test for If username or body is missing 
need another 404 test for if username doesn't exist